website/docs/guides/markdown-features/markdown-features-plugins.mdx

```diff
@@ -160,7 +160,7 @@ The writeup below is **not** meant to be a comprehensive guide to creating a plu
 For example, let's make a plugin that visits every `h2` heading and adds a `Section X. ` prefix. First, create your plugin source file anywhere—you can even publish it as a separate npm package and install it like explained above. We would put ours at `src/remark/section-prefix.js`. A remark/rehype plugin is just a function that receives the `options` and returns a `transformer` that operates on the AST.
 
 ```js "src/remark/section-prefix.js"
-import visit from 'unist-util-visit';
+import { visit } from 'unist-util-visit';
```

Comment:
```suggestion
import {visit} from 'unist-util-visit';
```

Reply:
Good! I was very conflicted because I didn't know which style you guys preferred. 😄